setup.py

```diff
@@ -142,7 +142,7 @@ def main():
         params = build_params()
 
     setup(
-        name='yt-dlp',
+        name='yt-dlp',  # package name (do not change/remove comment)
```

Comment:
What is the point of this? Perhaps that should be included within the comment?

Reply:
So we can change the package name programatically by searching for this comment.